src/main/java/frc/robot/subsystems/simulated/SimulatedSolenoid.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.simulated;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.simulation.SolenoidSim;
+
+
+/*
+ * A wrapper for a solenoid that echoes its settings back along get().
+ */
+public class SimulatedSolenoid extends Solenoid {
```

Comment:
Turns out the solenoid, when running in the simulator, doesn't copy its value from set to get; it expects the user to change the value by hand using the simulated solenoid board.

Wrapped the solenoid to copy its sets to a state that get can read.